### PR TITLE
chore(build): experimental Docker support for our build

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# EXPERIMENTAL Docker support for angular2 build process.
+# Build with: docker build -t $USER/angular:$(git rev-list -n 1 HEAD) .
+FROM node:5.7.1
+MAINTAINER Alex Eagle
+# Copy only the shrinkwrapped dependencies
+COPY npm-shrinkwrap.json /home
```
Comment (IgorMinar) [inline Dockerfile]: Use .clean.json please

**Comment by IgorMinar** [comment]
Lgtm otherwise

File: Dockerfile (hunk 1 above)
Comment (alexeagle) [inline Dockerfile]: yes, that reduces the changes seen, but need to copy to npm-shrinkwrap.json so npm install will see it, right?

**Comment by alexeagle** [comment]
Will leave this work on the docker branch, and not merge to master yet.

**Pull request closed without merging**